apps/address-service/domains/common/utils/services/InjectionsSeeker.js

```diff
@@ -0,0 +1,194 @@
+const { AddressInjection } = require('@address-service/domains/address/utils/serverSchema')
+const get = require('lodash/get')
+const { INJECTIONS_PROVIDER } = require('@address-service/domains/common/constants/providers')
+
+const SPECIAL_SYMBOLS_REGEX = /[!@#$%^&*)(+=.,\-_:;"'`[\]]/g
+
+/**
+ * A class used to search injections within database.
```

Comment:
```suggestion
 * Searches injections within database.
```
```suggestion
 * A class used to search injections within database.
```

Reply:
Class is not searching itself :) This is just a set of methods convenient to use and for testing.